go/chat/helper.go

```diff
@@ -97,24 +97,14 @@ func (s *sendHelper) conversation(ctx context.Context) error {
 		return err
 	}
 	uid := gregor1.UID(kuid.ToBytes())
-
-	oneChatPerTLF := true
-	convs, _, err := FindConversations(ctx, s.G(), s.DebugLabeler, s.remoteInterface, uid, s.canonicalName,
-		chat1.TopicType_CHAT, s.membersType, chat1.TLFVisibility_PRIVATE, s.topicName, &oneChatPerTLF)
+	conv, _, err := NewConversation(ctx, s.G(), uid, s.canonicalName, s.topicName,
```

Comment:
Probably not, `FindConversations` will use the local inbox cache do its thing, so if we know about the conversation at all we'll be able to know it exists without having to touch the server.